$NetBSD: patch-src_plugins_Makefile,v 1.1 2016/03/22 11:29:11 khorben Exp $

Build and link the "smscrypt" plug-in with OpenSSL as required.

--- src/plugins/Makefile.orig	2016-03-22 11:22:04.000000000 +0000
+++ src/plugins/Makefile
@@ -115,8 +115,8 @@ $(OBJDIR)profiles$(SOEXT): $(profiles_OB
 	$(CCSHARED) -o $(OBJDIR)profiles$(SOEXT) $(profiles_OBJS) $(profiles_LDFLAGS)
 
 smscrypt_OBJS = $(OBJDIR)smscrypt.o
-smscrypt_CFLAGS = $(CPPFLAGSF) $(CPPFLAGS) $(CFLAGSF) $(CFLAGS)
-smscrypt_LDFLAGS = $(LDFLAGSF) $(LDFLAGS)
+smscrypt_CFLAGS = $(CPPFLAGSF) $(CPPFLAGS) $(CFLAGSF) $(CFLAGS) `pkg-config --cflags openssl`
+smscrypt_LDFLAGS = $(LDFLAGSF) $(LDFLAGS) `pkg-config --libs openssl`
 
 $(OBJDIR)smscrypt$(SOEXT): $(smscrypt_OBJS)
 	$(CCSHARED) -o $(OBJDIR)smscrypt$(SOEXT) $(smscrypt_OBJS) $(smscrypt_LDFLAGS)
